src/utils/file.utils.ts

```diff
@@ -0,0 +1,194 @@
+const DEFAULT_IMAGE_WIDTH = 2560;
+const DEFAULT_IMAGE_QUALITY = 0.9;
+const WORD_SIZE = 4;
+
+declare global {
+    interface Blob {
+        lastModifiedDate?: number;
+        name?: string;
+    }
+}
```

Comment:
```suggestion
  interface Blob {
        lastModifiedDate?: number;
        name?: string;
 }
```

Should not be a global. Doing it this way might override environment

Reply:
sure @yashim-deriv 
will extend the type instead